diff --git a/dist/commonjs/babel/import-plugin.js b/dist/commonjs/babel/import-plugin.js
index c5f49f3796263dd99431f6179a2d88d438240d01..efa869c87262e0f13826b876b972943c70202eaf 100644
--- a/dist/commonjs/babel/import-plugin.js
+++ b/dist/commonjs/babel/import-plugin.js
@@ -15,7 +15,7 @@ function _default({
   const thisModuleDist = (0, _path.resolve)(__dirname, "../../../dist");
   const thisModuleSrc = (0, _path.resolve)(__dirname, "../../../src");
   function isFromThisModule(filename) {
-    return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc);
+    return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc) || filename.includes(`${_path.sep}react-native-web${_path.sep}`);
   }
   return {
     name: "Rewrite react-native to react-native-css",
diff --git a/dist/commonjs/metro/resolver.js b/dist/commonjs/metro/resolver.js
index 314ffa133698239ffe1bbdab5efd576828ee2e7a..4abeb851603d96d7aba49c8e23e58b1a773fb296 100644
--- a/dist/commonjs/metro/resolver.js
+++ b/dist/commonjs/metro/resolver.js
@@ -10,7 +10,7 @@ var _allowedModules = require("../babel/allowedModules.js");
 const thisModuleDist = (0, _nodePath.resolve)(__dirname, "../../../dist");
 const thisModuleSrc = (0, _nodePath.resolve)(__dirname, "../../../src");
 function isFromThisModule(filename) {
-  return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc);
+  return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc) || filename.includes(`${_nodePath.sep}react-native-web${_nodePath.sep}`);
 }
 function nativeResolver(resolver, context, moduleName, platform) {
   const resolution = resolver(context, moduleName, platform);
diff --git a/dist/module/babel/import-plugin.js b/dist/module/babel/import-plugin.js
index 345305c1ab22272d79fbe45a3f08e8a5d03e87f3..fe98b06b614c49d4e9bc8d5193c171ec6b3fdc0c 100644
--- a/dist/module/babel/import-plugin.js
+++ b/dist/module/babel/import-plugin.js
@@ -1,6 +1,6 @@
 "use strict";
 
-import { resolve } from "path";
+import { resolve, sep } from "path";
 import { getInteropRequireDefaultSource } from "./helpers.js";
 import { handleReactNativeIdentifierRequire, handleReactNativeImport, handleReactNativeObjectPatternRequire } from "./react-native.js";
 import { handleReactNativeWebIdentifierRequire, handleReactNativeWebImport, handleReactNativeWebObjectPatternRequire } from "./react-native-web.js";
@@ -11,7 +11,7 @@ export default function ({
   const thisModuleDist = resolve(__dirname, "../../../dist");
   const thisModuleSrc = resolve(__dirname, "../../../src");
   function isFromThisModule(filename) {
-    return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc);
+    return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc) || filename.includes(`${sep}react-native-web${sep}`);
   }
   return {
     name: "Rewrite react-native to react-native-css",
diff --git a/dist/module/metro/resolver.js b/dist/module/metro/resolver.js
index 8d1c73b0d67600a2216653f80df26de59e791675..e0be0ffe21d52685589bf6cfc1cdfc56a42ed1cf 100644
--- a/dist/module/metro/resolver.js
+++ b/dist/module/metro/resolver.js
@@ -5,7 +5,7 @@ import { allowedModules } from "../babel/allowedModules.js";
 const thisModuleDist = resolve(__dirname, "../../../dist");
 const thisModuleSrc = resolve(__dirname, "../../../src");
 function isFromThisModule(filename) {
-  return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc);
+  return filename.startsWith(thisModuleDist) || filename.startsWith(thisModuleSrc) || filename.includes(`${sep}react-native-web${sep}`);
 }
 export function nativeResolver(resolver, context, moduleName, platform) {
   const resolution = resolver(context, moduleName, platform);
